frontend/src/services/AuthService.ts

```diff
@@ -45,7 +45,15 @@ const handlePostLogin = async () => {
         // This is to update the FamLoginUser for FamLoginUser.accesses.
         // For now team decided to grab user's access only when user login and may change later.
         await LoginUserState.cacheUserAccess();
-        await LoginUserState.requiresAcceptTermsCondition() && showTermsForAcceptance()
+
+        if (
+            LoginUserState.state.value.famLoginUser?.accesses &&
+            LoginUserState.state.value.famLoginUser.accesses.length > 0
+        ) {
```

Comment:
Hi Catherine, just curious, what endpoint call and what error was it from backend before this condition is added?
Is it the "validate_user_requires_accept_terms_and_conditions" endpoint?

Reply:
yes, and that's just because of a general security check, that only FAM user (at least one role in FAM) can call them  <img width="852" alt="Screen Shot 2024-07-05 at 2 50 28 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/e7ab94e5-2975-4584-bb3b-9cf4dec86d4f">